jira/client.py

```diff
@@ -3149,6 +3152,34 @@ def move_to_backlog(self, issue_keys):
             raise NotImplementedError('No API for moving issues to backlog for agile_rest_path="%s"' %
                                       self._options['agile_rest_path'])
 
+    def map_custom_python_name(self, customfieldname=None, pythonfieldname=None):
+        python_to_custom = self.field_map
+        custom_to_python = [(v,k) for k,v in python_to_custom]
+
+        try:
+            if customfieldname is not None:
+                return list_lookup(custom_to_python, customfieldname)
+            elif pythonfieldname is not None:
+                return list_lookup(python_to_custom, pythonfieldname)
+            else:
+                raise KeyError("Need to provide a custom field name, or a python field name.")
+        except:
```

Comment:
Naked except blocks are rarely a good idea... this is going to catch and hide any unexpected exception that may arise within the `try` block... (Including the one you're raising on the line above, which is probably not what you intended?)  👎